modules/rules_python/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "homepage": "https://github.com/bazelbuild/rules_python",
+    "maintainers": [
+        {
+            "email": "bazel-dev@googlegroups.com",
```

Comment:
this is wrong, the bazel team has not been participating **at all** in rules_python for the last year.

Maintainers are me, @hrfuller and @thundergolfer

How will you prevent this kind of mistake for future PRs to this repo? I've already tracked down the maintainers of all the rules as part of the rules authors SIG so I think we should work from that.

Reply:
Sorry, this is auto-generated by a script I wrote: https://github.com/bazelbuild/bazel-central-registry/blob/main/tools/add_module.py#L136 
I was actually thinking about asking if you want to maintain this. 
I guess fixing this script is the first step to avoid this kind of error...

Reply:
1. "Maintainer" here does *not* mean a contributor to the project itself, rather a person who looks after the *module definition* here in the registry. When a new version is submitted, or new patches applied, the maintainers should be notified and approve the request. That much can definitely be done by the Bazel team. With that said, if the original contributors of the project are actively interested in maintaining the Bazel modules (very likely for rulesets, very unlikely for C++ projects like zlib), they can definitely be added to the maintainer list.
2. The rule author SIG is definitely a good starting point to assign maintainers with active interest (though as noted elsewhere, the registry isn't for rulesets only).
3. The central registry is still in alpha phase. We haven't opened this up yet; this PR is just part of the process to check in some basic Bazel modules so that we can build Bazel itself with the new system. We fully intend to assign the maintainers to interested parties before the registry is opened up for public use.

Reply:
I think the concept you're looking for is who approves PRs against this file, and that belongs in a CODEOWNERS file. It's super confusing to put that metadata here, I think most ppl would be like me and assume that modules/rules_python/metadata.json is the metadata for the rules_python module

Reply:
Interesting, I didn't expect it to be so confusing.

Similar to https://packages.debian.org/sid/libgrpc++-dev, when you see the maintainer in this Debian package, do you think they must be the author of the project?

Maybe this can be clarified in documentation?

Reply:
Yeah I understand what you're getting at now. It's possible for someone other than the rules authors to own the registration of those rules. Do you think that would be common though? Won't rules authors have to come make a PR to this repo anyway every time they cut a release?

Reply:
https://github.com/bazelbuild/bazel-central-registry/tree/maintainer#module-maintainer

> Do you think that would be common though? 

It's totally possible for some non-Bazel projects. For example, many of TensorFlow's dependencies: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace2.bzl#L134

> Won't rules authors have to come make a PR to this repo anyway every time they cut a release?

That's right. For rules authors, they should definitely be the maintainer for their entries.